docs/devel/automation.md

```diff
@@ -108,6 +108,18 @@ with numerous other functions. See the README in the link above.
 Please feel free to unleash your creativity on this tool, send us new mungers
 that you think will help support the Kubernetes development process.
 
+### Closing stale pull-requests
+
+Github Munger will close pull-requests that don't have human activity in the
+last 90 days. It will warn about this process 60 days before closing the
+pull-request, and warn again 30 days later. One way to prevent this from
+happening is to add the "keep-open" label on the pull-request.
```

Comment:
Please point out that a comment on the PR will keep it open for another 30 days, as well.